FrameworkSystem/Client/ProxyManagerClient.py

```diff
@@ -610,10 +467,42 @@ def getUserProxiesInfo(self):
 
         :return: S_OK(dict)/S_ERROR()
     """
-    result = RPCClient("Framework/ProxyManager", timeout=120).getUserProxiesInfo()
+    result = RPCClient("Framework/ProxyManager", timeout=120, **self.__extArgs).getUserProxiesInfo()
     if 'rpcStub' in result:
       result.pop('rpcStub')
     return result
 
+  def deleteProxy(self, userDNs):
+    """ Delete proxy
+
+        :param list userDNs: user DNs
+
+        :return: S_OK()/S_ERROR()
+    """
+    return RPCClient("Framework/ProxyManager", timeout=120, **self.__extArgs).deleteProxyBundle(userDNs)
+
+  def getGroupsStatusByUsername(self, username, groups=None):
+    """ Get status of every group for DIRAC user:
+
+        :param str username: user name
+
+        :return: S_OK(dict)/S_ERROR()
+    """
+    # # { <user>: {
+    # #       <group>: [
+    # #         {
+    # #           Status: ..,
+    # #           Comment: ..,
+    # #           DN: ..,
+    # #           Action: {
+    # #             <fn>: { <opns> }
+    # #           }
+    # #         },
+    # #         { ... }
+    # #       ],
+    # #       <group2>: [ ... ]
+    # # } }
+    rpc = RPCClient("Framework/ProxyManager", timeout=120, **self.__extArgs)
+    return rpc.getGroupsStatusByUsername(username, groups)
```

Comment:
Same as for previous comment. This function is not needed.

Reply:
fixed